packages/rabbitmq/src/rabbitmq.module.ts

```diff
@@ -100,28 +96,6 @@ export class RabbitMQModule
     return connection;
   }
 
-  public static build(config: RabbitMQConfig): DynamicModule {
```

Comment:
It was advised years ago to adopt `forRoot`  which is the common nestjs way therefore this is expected if anyone is using it